class/Defaults/Carrier/Webhook.php

```diff
@@ -226,12 +187,13 @@ public function send_post( $url, $args = [], $headers = [] ) {
 			$remote_args,
 			$this,
 		], '6.0.0', 'notification/carrier/webhook/called/post' );
+
 		do_action( 'notification/carrier/webhook/called/post', $response, $url, $args, $remote_args, $this );
```

Comment:
```suggestion
		do_action( 'notification/carrier/webhook/called', $response, $url, $method, $args, $remote_args, $this );
```

Make this action more elastic. Make sure to put the old action as deprecated.